### PR TITLE
KUI-1388: Updated api to handle recommended prerequisites

### PR DESCRIPTION
**Pull request by axelbjo**

### CODE REVIEW

File: scripts/importData/index.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ const csvConfig = {
     'courseCode',
     'courseDisposition_sv',
     'courseDisposition_en',
+    'recommendedPrerequisites_sv',
+    'recommendedPrerequisites_en',
```
Comment (karlandindrakryggen) [inline scripts/importData/index.js:13]: I'm not sure, but I think we need to create a new import script for just recommendedPrerequisites_sv/en. But for now I think we can wait with that, ie not updating importData/index.js and importData/index2.js at all

**Review by karlandindrakryggen — CHANGES_REQUESTED** [review]
Look into the failing integration test and revert updates in import script.

**Review by karlandindrakryggen — APPROVED** [review]

**Comment by karlandindrakryggen** [comment]
Remember to not merge this for now (needs to be synced with release of changes in other repos)

File: docker-compose-integration-tests.yml
Comment (axelbjo) [inline docker-compose-integration-tests.yml:1]: This is deprecated and is not needed any longer

https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313

**Review by amirhossein-haerian — APPROVED** [review]
LGTM, Great Job ⭐️

**Review by allazis — APPROVED** [review]
Good job!

**Pull request merged**